client/src/components/map/legend/index.tsx

```diff
@@ -2,37 +2,37 @@ import React, { useMemo, Children, isValidElement } from 'react';
 
 import { cn } from '@/lib/classnames';
 
-import SortableList from './sortable/list';
+// import SortableList from './sortable/list';
 import { LegendProps } from './types';
 
 export const Legend: React.FC<LegendProps> = ({
   children,
   className = '',
-  sortable,
-  onChangeOrder,
-}: LegendProps) => {
+}: // sortable,
```

Comment:
You don't need to comment these props. If you don't want to use the sortable it will be as simple as passing enabled false to the Legend. By doing this you'll keep all the functionalities you had before. 
```jsx
sortable={{
  enabled: false
}}
```